tests/e2e-evm/test/helpers/abi.ts

```diff
@@ -0,0 +1,21 @@
+import { Abi, AbiFallback, AbiReceive } from "abitype";
+
+export function getAbiFallbackFunction(abi: Abi): AbiFallback | undefined {
```

Comment:
ha such a golang guy rocking these for-loops

would recommend a single helper func like (i've converted to functional, keep your for loops if ya want 😄)
```ts
export const getAbiFunctionByType = (abi: Abi, funcType: string) => abi.find(f => f.type === funcName)
```

if you want to have the same funcs as here you can convert them to use that:
```ts
const getAbiFallbackFunction = (abi: Abi) => getAbiFunctionByType(abi, "fallback")
const getAbiReceiveFunction = (abi: Abi) => getAbiFunctionByType(abi, "receive")
```

Reply:
+1, I did a pretty fast extraction of those helpers and should be leveraging these other approaches